### DISCUSSION TITLE
Epiverse-TRACE developer space – Welcome to Epiverse-TRACE

### DISCUSSION THREAD

**Discussion by giscus** [epiverse-trace/epiverse-trace.github.io#259] [Blog post discussion]
# Epiverse-TRACE developer space – Welcome to Epiverse-TRACE

A place for Epiverse-TRACE developers to share their reflections, learnings, and showcase their work.

https://epiverse-trace.github.io/

**Comment by paulkorir** [comment]
It might be helpful to add the following somewhere prominent to help users easily install Epiverse packages:

```
# install the 'universe' package
install.packages("universe", repos="https://ropensci.r-universe.dev")
# add the 'epiverse-trace' repo
universe::add("epiverse-trace")
# install a package e.g. 'episoap'
install.packages("episoap")
```
